awd/cli.py

```diff
@@ -185,30 +186,27 @@ def deposit(
 
             # Retrieve and validate Crossref metadata
             crossref_response = crossref.get_response_from_doi(metadata_url, doi)
-            if crossref.is_valid_response(doi, crossref_response) is False:
+            if article.valid_crossref_metadata(crossref_response) is False:
                 continue
-            article.crossref_metadata = crossref.get_metadata_extract_from(
-                crossref_response.json()
-            )
+            article.crossref_metadata = crossref_response.json()
```

Comment:
**UPDATE**: please feel free to disregard if planned for movement to `.process()`.

Now that this logic has moved to an `Article `method, I'd suggest that this business logic should be handled by the class itself.  While poking around in the `Article` class, it was jarring that I had to jump back to the CLI to see how the property `self.crossref_metadata` was set.

With some updates in `Article`, I could see simplifying down to (and tipping towards a higher level `.process()` method):

```python
# Retrieve and validate Crossref metadata
try:
    article.retrieve_crossref_metadata(metadata_url)
except InvalidCrossrefMetadata:
    continue
```

Then in `Article` you'd need something like:

```python

# custom exception that can be handled in CLI
class InvalidCrossrefMetadata(Exception):
   pass

class Article:
# ...
# ...
def retrieve_crossref_metadata(self, metadata_url) -> None:
    
    crossref_response = crossref.get_response_from_doi(metadata_url, doi)        
    if self.valid_crossref_metadata(crossref_response) is False:
        raise InvalidCrossrefMetadata("invalid metadata from crossref") # raises custom exception to be handled
    self.crossref_metadata = crossref_response.json()
```